src/Autofac.Multitenant/MultitenantContainer.cs

```diff
@@ -394,25 +404,43 @@ private ILifetimeScope CreateTenantScope(object tenantId, Action<ContainerBuilde
     /// <returns><c>true</c> if an existing configuration was removed; otherwise, <c>false</c>.</returns>
     /// <seealso cref="ConfigurationActionBuilder"/>
     /// <seealso cref="ConfigureTenant(object, Action{ContainerBuilder})"/>
+    [SuppressMessage("CA1513", "CA1513", Justification = "ObjectDisposedException.ThrowIf is not available in all target frameworks.")]
     public bool ReconfigureTenant(object tenantId, Action<ContainerBuilder> configuration)
     {
         if (configuration == null)
         {
             throw new ArgumentNullException(nameof(configuration));
         }
 
+        if (_isDisposed == 1)
```

Comment:
Might be worth a unit test to try reconfiguring a tenant after disposal - so we ensure we don't regress that later.

Reply:
ok